charts/kubeai/values.yaml

```diff
@@ -101,10 +101,7 @@ fullnameOverride: "kubeai"
 command:
 - /app/manager
 
-args:
-- --metrics-bind-address=:8080
-- --leader-elect
-- --health-probe-bind-address=:8081
+args: []
```

Comment:
can you separate the helm changes in a separate PR? This will allow us to first publish a new version of the image and then update appVersion and helm changes at the same time. This prevents possibility of incompatibility.

Reply:
Talked about this offline